CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-validation ChangeLog
 
+## 7.1.0 -
+
+### Added
+- Added a validator for `VerifiablePresentation`
+- Added a validator for `VerifiableCredential`.
+- Added a validator for `jsonldIdentifier`.
```

Comment:
Hmm, I don't think we want a "JSON-LD identifier"... such a value is just a URI.

Looking below at the implementation, I see that what you're going for, but it isn't an identifier, but rather a "subject or object node that may have an aliased `id` in a JSON-LD representation of a graph". People will not get that from the name "jsonldIdentifier".

So this validator is really just an identifier or an optionally-identified object. I don't know how useful this validator will actually be for general purpose -- because you often want to validate other properties too. This also isn't the only way to represent an object in JSON-LD, it's specific to a particular use case.

So the validator is really an "IdentifierOrObjectWithIdentifier" ... which, of course, is a terrible name. I'm trying to think of what to do here. We could rename this to `optionallyIdentifiedObject` and make the `id` so it isn't required... hmm.

I recommend that we just copy the internals of this for now and not expose a separate schema for public consumption that will confuse people. I can't think of a better solution at the moment.

Reply:
I created a new dir `schemas/helpers` which should not be publicly exported via `validation.schemas`. I also renamed it it to `idOrObjectWithId.js` to remove any association with linked data: https://github.com/digitalbazaar/bedrock-validation/pull/44/commits/bc0002965efb6454c48044f1864eb6badfe33341

LMK if that solution work. If it does then we have a way of creating validators that we don't want to export publicly.